frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -199,6 +208,53 @@ const FormConceptGroup = (props: Props) => {
             ? t("externalForms.common.concept.copying")
             : props.attributeDropzoneText
         }
+        dropBetween={(i: number) => {
+          return (item: PossibleDroppableObject) => {
+            if (item.type !== DNDType.CONCEPT_TREE_NODE) return;
+
+            if (props.isValidConcept && !props.isValidConcept(item))
+              return null;
+
+            const concept = isMovedObject(item)
+              ? copyConcept(item)
+              : initializeConcept(item, defaults, tableConfig);
+            let newPropsValue = props.value;
+            let insertIndex = i;
+            if (isMovedObject(item)) {
+              const { movedFromFieldName, movedFromAndIdx, movedFromOrIdx } =
+                item.dragContext;
+
+              if (movedFromFieldName === props.fieldName) {
+                const willConceptMoveUp =
```

Comment:
Shouldn't it be called `willConceptMoveDown`? The dragged concept is moving down, no?